evap/staff/tests/test_live.py

```diff
@@ -0,0 +1,111 @@
+from selenium.webdriver.support.wait import WebDriverWait
+
+from evap.evaluation.models import Contribution, Course, Degree, Evaluation, Question, Questionnaire, UserProfile
+from evap.evaluation.tests.tools import LiveServerTest, make_manager
+
+
+class StaffLiveTests(LiveServerTest):
+    def test_copy_header(self):
+        self._default_login()
+
+        self._enter_staff_mode()
+
+        self.selenium.get(self.live_server_url + reverse("staff:user_import"))
+
+        WebDriverWait(self.selenium, 10).until(
+            expected_conditions.visibility_of_element_located((By.CLASS_NAME, "btn-link"))
+        )
+
+        # Patch clipboard functions to test functionality
+        self.selenium.execute_script("navigator.clipboardExecuted = false;")
+        self.selenium.execute_script("navigator.clipboard.writeText = (c) => navigator.clipboardExecuted = c;")
+
+        self.selenium.find_element(By.CLASS_NAME, "btn-link").click()
+
+        copied_text = self.selenium.execute_script("return navigator.clipboardExecuted;")
+        self.assertEqual(copied_text, "Title\tFirst name\tLast name\tEmail")
+
+    def test_changes_form_data(self):
+        manager = make_manager()
+        responsible = baker.make(UserProfile)
+        editor = baker.make(UserProfile)
+        evaluation = baker.make(
+            Evaluation,
+            course=baker.make(Course, degrees=[baker.make(Degree)], responsibles=[responsible]),
+            vote_start_datetime=datetime(2099, 1, 1, 0, 0),
+            vote_end_date=date(2099, 12, 31),
+        )
+
+        baker.make(Questionnaire, questions=[baker.make(Question)])
+        general_question = baker.make(Question)
+        general_questionnaire = baker.make(Questionnaire, questions=[general_question])
+        evaluation.general_contribution.questionnaires.set([general_questionnaire])
+        contributor_question = baker.make(Question)
+        contributor_questionnaire = baker.make(
+            Questionnaire,
+            type=Questionnaire.Type.CONTRIBUTOR,
+            questions=[contributor_question],
+        )
+        contribution1 = baker.make(
+            Contribution,
+            evaluation=evaluation,
+            contributor=responsible,
+            order=0,
+            role=Contribution.Role.EDITOR,
+            textanswer_visibility=Contribution.TextAnswerVisibility.GENERAL_TEXTANSWERS,
+        )
+        contribution2 = baker.make(
+            Contribution,
+            evaluation=evaluation,
+            contributor=editor,
+            order=1,
+            role=Contribution.Role.EDITOR,
+        )
+        contribution1.questionnaires.set([contributor_questionnaire])
+        contribution2.questionnaires.set([contributor_questionnaire])
+
+        self._login(manager)
+
+        self._enter_staff_mode()
+
+        self.selenium.get(self.live_server_url + reverse("staff:evaluation_edit", args=[evaluation.pk]))
+
+        self._screenshot("changes_form_data")
```

Comment:
leftover(?) screenshot command -- we don't currently do anything with this, right?